node/node/config.go

```diff
@@ -13,8 +13,8 @@ import (
 	"github.com/celestiaorg/celestia-node/service/rpc"
 )
 
-// ConfigLoader defines a function that loads a config from any source.
-type ConfigLoader func() (*Config, error)
```

Comment:
This seems to be an artifact from times where the pkg was called `config`

Reply:
removed